src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__conteiner">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+<<<<<<< HEAD
+        <h3 class="card__discription">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+=======
+        <h2 class="card__discription">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+>>>>>>> 12cda43590362e9157c7c52ff2a94b97ed3ce4fa
```

Comment:
Now you have a merge conflict that needs to be resolved because it also affects how your app looks visually.
![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/a84897b5-0e55-4ca5-b032-a975695e96c0)